operations/deployment/terraform/certificates.tf

```diff
@@ -1,7 +1,7 @@
 # Lookup for main domain.
 
 data "aws_acm_certificate" "issued" {
-  count  = var.no_cert == true ? 0 : (var.create_root_cert != "true" ? (var.create_sub_cert != "true" ? (local.fqdn_provided ? 1 : 0) : 0) : 0)
+  count  = var.no_cert == "true" ? 0 : (var.create_root_cert != "true" ? (var.create_sub_cert != "true" ? (local.fqdn_provided ? 1 : 0) : 0) : 0)
```

Comment:
Booleans should be treated as Booleans. 
Yes we could do a string check but when possible let's use the type

Reply:
Thing is that there are some variables being passed from the action itself. To avoid type errors, treating them as strings. 
Just to be consistent with how we treat them, reverting to string. This same line would have booleans and strings if not.